### ISSUE TITLE
migrating datastore procedure does not discuss services instances

### ISSUE THREAD

**Issue by randomtask1155** [pivotal-cf/docs-pcf-refarch#67]
https://github.com/pivotal-cf/docs-pcf-refarch/blob/b8af6db1e158b30225256a02282c5739f00a67b3/vsphere/vsphere_migrate_datastore.html.md.erb#L99

see community thread 
https://community.pivotal.io/s/question/0D50e00006WQPvU/change-vsphere-datastore-recreate-bosh-vms-what-needs-to-be-done?language=en_US&s1oid=00DU0000000YQaz&s1nid=0DB0P000000PDKg&emkind=chatterCommentNotification&s1uid=0050P000007YjIk&emtm=1594635676051&fromEmail=1&s1ext=0

The doc should say recreate errands should be enabled for on demand tiles.  if recreate errand does not exist then services instances will need to be recreated manually, Only after a successful run of "upgrade all service instances" errand, using the the procedure outlined in the community post.

**Comment by mlimonczenko** [comment]
Hello @randomtask1155,

We were unable to prioritize this request at the time the issue was filed.

If this issue is still relevant, submit a new pull request (preferred) or a new GitHub issue.

I am closing this request. Thank you so much for your contribution.
